vault/logical_system_paths.go

```diff
@@ -224,6 +498,20 @@ func (b *SystemBackend) auditPaths() []*framework.Path {
 				logical.DeleteOperation: b.handleAuditedHeaderDelete,
 				logical.ReadOperation:   b.handleAuditedHeaderRead,
 			},
+			Operations: map[logical.Operation]framework.OperationHandler{
```

Comment:
Can we remove the `Callbacks` field above?

Reply:
Yep! 👍